omero/developers/Modules/Search.rst

```diff
@@ -57,10 +61,20 @@ FullTextAnalyzer <src/main/java/ome/services/fulltext/FullTextAnalyzer.java>`.
 
 Assuming these entries above for Image.name:
 
--  searching for **GFP-H2B** returns 1 and 2.
--  searching for **"GFP H2B"** also returns 1 and 2.
--  searching for **GFP H2B** returns 1, 2, and 3, since the two terms
+-  searching for **GFP-H2B** returns 1, 2, 3 and 4, because of the tokenizing on the hyphen and joining of the tokens by an **OR**.
+-  searching for **"GFP H2B"** only returns 1 and 2, since the quotes enforce the exact sequence of the tokens and the query is built with an **AND**.
+-  searching for **GFP H2B** returns 1, 2, 3 and 4, since the two tokens
    are joined by an **OR**.
+-  searching for **"GFP-H2B"** returns 1 and 2.
+
+With the same entries as above and adding a wildcard:
+
+-  searching for **\*FP** returns 1, 2, 3 and 4. As this example shows, **leading wildcards in the Graphical User Interface are allowed**, but disallowed when using the API directly, see below in the developers section.
+-  searching for **GF\*** returns 1, 2, 3 and 4.
+-  searching for **GFP-\*** returns no results, but **GFP.\*** returns 1, 2, 3 and 4. Only hyphen and underscore do not return results in this situation, the other non-alpha-numerical characters do.
+-  searching for **"\*FP-H2B"** returns no results. This is not a wildcard query, but a term query, and is the same as searching for **"FP-H2B"**.
```

Comment:
I really like these, but they might get their own sections that just explains, "a wildcard in quotes doesn't count"

Reply:
Surprising new result. 
``"GFP*H2B"`` does return 1 and 2. So more precisely, wildcard in quotes does not count as wildcard, but as a non-alphanumerical character on which to tokenize. (?)

Reply:
I think that explanation makes a lot of sense.